make_templates.py

```diff
@@ -25,7 +25,12 @@ def get_env() -> Dict[str, Any]:
         open('all-hooks.json').read(),
         object_pairs_hook=collections.OrderedDict,
     )
-    return {'all_hooks': all_hooks}
+    all_types = {
+        hook_type
+        for properties in all_hooks.values()
+        for hook_type in properties[0].get("types", [])
```

Comment:
there's also `types_or` now, not sure how this will filter 🤔

Reply:
i joined `types` and `types_or` now.